draft-ietf-quic-http.md

```diff
@@ -536,9 +536,10 @@ HTTP/3 request header; when it is, it MUST NOT contain any value other than
 
 This means that an intermediary transforming an HTTP/1.x message to HTTP/3 will
 need to remove any fields nominated by the Connection field, along with the
```

Comment:
The "will need to" here now reads like RFC 6919 language.  Why not MUST?

Reply:
Works for me.